src/praisonai-agents/praisonaiagents/tools/github_tools.py

```
@@ -136,7 +136,7 @@ def github_create_branch(branch_name: str) -> str:
         # Validate branch name to prevent misinterpretation as git options or invalid refs
         try:
             subprocess.run(
-                ["git", "check-ref-format", "--branch", branch_name],
+                ["git", "check-ref-format", f"refs/heads/{branch_name}"],
```

Ruff (0.16.3)

[error] Starting a process with a partial executable path

(S607)